packages/FormElement/src/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [0.3.0] - 2020-01-17
```

Comment:
I think you should also say what was removed (https://github.com/acl-services/paprika/pull/371/files#diff-70850fa02375fbeb5f7256649db70099L167) since that's the breaking change to the API.